### ISSUE TITLE
Why are you enqueuing the scripts on the front end?

### ISSUE THREAD

**Issue by plexus77** [mattkeys/ACF-Conditional-Taxonomy-Rules#11]
I noticed that you are enqueuing the `includes/input_conditional_taxonomy.js` with the frontend scripts.  Maybe I have missed something but I don't think this is needed and is adding unnecessary overhead.

**Comment by mattkeys** [comment]
ACF has a frontend component, read more here: https://www.advancedcustomfields.com/resources/acf_form/

The enqueue has the 'acf-input' as a dependency so this plugin should only add the script to the frontend in situations where ACF is already being added to the frontend as well. But if your project will never use this on the front-end or if it is giving you issues, feel free to remove it in your installation.

**Comment by plexus77** [comment]
Hey Matt, thanks for the quick response... Yeah I see that, however when I comment out that line (I had already done this) all of the other acf dependencies disappear as well.  So it seem that it was your plugin that was pulling in the ACF-input and jQuery UI JS dependencies.

**Comment by mattkeys** [comment]
Definitely possible. I don't think it was at the time of writing this, but if ACF has been changed in more recent versions so that the script is registered, but not enqueued at the time this plugin does its own script enqueue, then it would enqueue the ACF scripts along with this one.

The project I originally wrote this for uses ACF on the frontend, but looking at that code I see I am doing something like this in the enqueue:

`		if ( wp_script_is( 'acf-input', 'enqueued' ) ) {
			wp_enqueue_script( 'acf-input-conditional-taxonomy', ACFCTR_PUBLIC_PATH . 'includes/input_conditional_taxonomy.js', array( 'acf-input' ), '3.0.0' );
		}`

If I recall I didn't include that logic in the version of this plugin listed here because I was seeing some issues with this sometimes registering our custom conditionals due to a timing issue re: when ACF looks for registered conditionals (it was sometimes registered too late). I reported the issue with the timing of registering conditionals to Elliot so he is aware and it is possible that it has been fixed by now. For what it is worth I also pointed him to this plugin and he said he was planning to incorporate conditional taxonomies into the main plugin as well.
